### DISCUSSION TITLE
115网盘文件受限

### DISCUSSION THREAD

**Discussion by szshaojiang** [AlistGo/alist#3935] [Q&A]
win10+radrive环境，管理员权限挂载，文本文件可以上传下载也可以直接打开编辑，图片、视频打不开报没有权限，
下载报错
![image](https://user-images.githubusercontent.com/23231380/227405928-c90694ab-258c-4cde-8c3b-44fb6b776243.png)，
要s-0-1-1权限。
图片上传可以但很卡

**Comment by smarxtboy** [comment]
2023年3月23日，使用Alist挂载115网盘，发现之前没有出现的问题是：成功挂载后，网盘的资源无法下载，可以上传。目前115网盘自身的客户端没有出现这样的问题。其次这个问题导致配合Infuse 已经无法观看网盘任何电影电视剧。官方回答是不支持第三方挂载。

**Reply by WUFXXX** [reply]
我更新到3.14版本也出现这个问题，老版本正常使用

**Comment by wangzixin1** [comment]
115网盘无法下载

**Reply by milipapa** [reply]
3.4.0docker可以下载，不能上传
